### PR TITLE
Master voip fwi

### PR DESCRIPTION
**Pull request by fwintjens**
https://www.odoo.com/web?#id=6967&view_type=form&model=project.task&action=333&active_id=248

### CODE REVIEW

File: addons/crm/crm_phonecall.py [hunk 1]
```diff
@@ -33,48 +33,37 @@ class crm_phonecall(osv.osv):
...
-            return context.get('default_state')
-        return 'open'
-
     _defaults = {
         'date': fields.datetime.now,
         'priority': '1',
-        'state':  _get_default_state,
+        'state':  'to_do',
         'user_id': lambda self, cr, uid, ctx: uid,
         'active': 1
```
Comment (KangOl) [inline addons/crm/crm_phonecall.py]: remove default value if you remove the field.

File: addons/crm/wizard/crm_phonecall_to_phonecall.py [hunk 2]
```diff
@@ -59,18 +58,20 @@ def action_schedule(self, cr, uid, ids, context=None):
                     this.user_id and this.user_id.id or False, \
                     this.team_id and this.team_id.id or False, \
                     this.categ_id and this.categ_id.id or False, \
-                    action=this.action, context=context)
-
-        return phonecall.redirect_phonecall_view(cr, uid, phocall_ids[phonecall_ids[0]], context=context)
+                    context=context)
+        return {
+            'type': 'ir.actions.client',
+            'tag': 'reload_panel',
```
Comment (KangOl) [inline addons/crm/wizard/crm_phonecall_to_phonecall.py]: Dependency problem.
Will not works with only `crm` module installed.

File: addons/crm_voip/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "crm_voip",
+
+    'summary': """
+        Short (1 phrase/line) summary of the module's purpose, used as
+        subtitle on modules listing or apps.openerp.com""",
+
+    'description': """
+        Long description of module's purpose
```
Comment (KangOl) [inline addons/crm_voip/__openerp__.py]: better description needed.

File: addons/crm_voip/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    """,
+
+    'author': "OpenERP SA",
```
Comment (KangOl) [inline addons/crm_voip/__openerp__.py]: Odoo SA

File: addons/crm_voip/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    'website': "http://www.odoo.com",
```
Comment (KangOl) [inline addons/crm_voip/__openerp__.py]: https

**Comment by christophlsa** [comment]
Nice work. Is it possible to use this with a normal SIP Account without a telephone installation? In the settings I can set a socket. Which socket is meant with this setting?

**Pull request closed without merging**